argoverse/data_loading/synchronization_database.py

```diff
@@ -62,23 +90,50 @@ def __init__(self, dataset_dir: str, collect_single_log_id: Optional[str] = None
         else:
             log_fpaths = [f"{dataset_dir}/{collect_single_log_id}"]
 
-        self.per_log_pose_index: Dict[str, Dict[str, np.ndarray]] = {}
+        self.per_log_camtimestamps_index: Dict[str, Dict[str, np.ndarray]] = {}
+        self.per_log_lidartimestamps_index: Dict[str, np.ndarray] = {}
 
         for log_fpath in log_fpaths:
-            log_id = log_fpath.split("/")[-1]
+            log_id = Path(log_fpath).name
 
-            self.per_log_pose_index[log_id] = {}
+            self.per_log_camtimestamps_index[log_id] = {}
             for camera_name in STEREO_CAMERA_LIST + RING_CAMERA_LIST:
 
                 sensor_folder_wildcard = f"{dataset_dir}/{log_id}/{camera_name}/{camera_name}_*.jpg"
-                tovs = get_timestamps_from_sensor_folder(sensor_folder_wildcard)
+                cam_timestamps = get_timestamps_from_sensor_folder(sensor_folder_wildcard)
+
+                self.per_log_camtimestamps_index[log_id][camera_name] = cam_timestamps
 
-                self.per_log_pose_index[log_id][camera_name] = tovs
+            sensor_folder_wildcard = f"{dataset_dir}/{log_id}/lidar/PC_*.ply"
+            lidar_timestamps = get_timestamps_from_sensor_folder(sensor_folder_wildcard)
+            self.per_log_lidartimestamps_index[log_id] = lidar_timestamps
 
     def get_valid_logs(self) -> Iterable[str]:
         """ Return the log_ids for which the SynchronizationDatabase contains pose information.
         """
-        return self.per_log_pose_index.keys()
+        return self.per_log_camtimestamps_index.keys()
+
+    def get_closest_lidar_timestamp(self, cam_timestamp: int, log_id: str) -> Optional[int]:
+        """
```

Comment:
Docstring?

Reply:
@mgilson-argo thanks for catching that, just added it